comms/dht/src/store_forward/forward.rs

```diff
@@ -189,6 +190,19 @@ where S: Service<DecryptedDhtMessage, Response = (), Error = PipelineError>
             return Ok(());
         }
 
+        if let Some(expires) = &dht_header.expires {
```

Comment:
What do we do if it does not have an expires on it?
Currently, it seems that if you want your saf messages to live longer, which is something a wallet will want to do, you can just leave this out and the messages wont expire? As they will be forwarded constantly?

Should we not add a expire timestamp here before we handle it further?

Reply:
Currently treat it as if it had no expiry, for backwards compatibility.

Reply:
There will be a separate PR to add the field to the signature, though this will be a breaking change as previously mentioned in the above comments.